tait.py

```diff
@@ -218,11 +218,13 @@ def ccr_set_rx_ctcss(self, ctcss_tone_freq_hz):
 
     '''
     Send raw commands to the radio. Assumes the arg is a string (you must convert in advance).
-    
+
     At least one of the author's Tait radios has a nasty habit of not responding sometimes. So, we're going to do retries.
+    (We can't do exponential retries because modifying the read timeout requires closing/re-opening the serial device)
```

Comment:
I removed this comment in my copy of the code, must have left it in the on-box copy. 

It's not true since there's a sleep after read (rather than during)

Reply:
Removed.